### PR TITLE
One pager updates

### PR DESCRIPTION
**Pull request by stdavis**
Pull request description
------------------------

### CODE REVIEW

File: data/bioscience-overview/index.html [hunk 1]
```diff
@@ -6,7 +6,6 @@
   email: jpechmann@utah.gov
 tags:
   - bioscience
-  - data
   - fire
   - gap landcover
   - habitat
```
Comment (steveoh) [inline data/bioscience-overview/index.html]: please add the `overview` tag

File: data/bioscience-overview/index.html [hunk 2]
```diff
@@ -17,66 +16,66 @@
...
+      <td><a href="https://drive.google.com/drive/folders/0ByStJjVZ7c7mNEFsbWhudnI3SHc">Dominant Vegetation</a></td>
+      <td>map data</td>
+      <td>Utah Division of Wildlife Resources (DWR)</td>
+      <td>Statewide distribution of dominant vegetation species in Utah. The data were derived from 1:24,000 scale orthophoto quadrangles, 1:40,000 scale aerial photos and field mapping by the Utah Division of Wildlife Resources (approximate date Jan. 2001).</td>
+    </tr>
+    <tr>
+      <td>GAP Landcover Classification (Raster)</td>
+      <td>map data and map services</td>
+      <td>USGS GAP Analysis Program, USU RS/GIS Lab</td>
+      <td>Landcover classification for the State of Utah for the purposes of regional terrestrial biodiversity assessment.<br /><a href="http://earth.gis.usu.edu/swgap/landcover_download.html"> Download data files</a><br />
```
Comment (steveoh) [inline data/bioscience-overview/index.html]: please remove the `br`'s

Comment (stdavis) [inline data/bioscience-overview/index.html]: I actually thought that they were warranted...
with:
image
without:
image

But you are the website czar. So let me know what you think.

Comment (steveoh) [inline data/bioscience-overview/index.html]: Ok. Ignore that then.

**Comment by stdavis** [comment]
> Did you update the stewardship table in drive?

Thanks for the reminder. I just did and added it to the steps.

**Comment by steveoh** [comment]
Thanks for the help!

**Pull request merged**